test/two_finger_grasp_filter_test.test

```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <launch>
-    <include file="$(find panda_moveit_config)/launch/planning_context.launch">
-      <arg name="load_robot_description" value="true"/>
+    <!-- Suction or two finger -->
+    <arg name="gripper" value="two_finger" doc="must be one of 'suction' or 'two_finger'" />
```

Comment:
You set the value, why have the doc string?

Reply:
documentation